File: winter-compiler/src/test/resources/WithInjectedField_WinterMembersInjector.java
package test;

import io.jentz.winter.ClassTypeKey;
import io.jentz.winter.Graph;
import io.jentz.winter.inject.MembersInjector;
import java.lang.Override;
import java.lang.String;
import javax.annotation.Generated;

@Generated(
        value = "io.jentz.winter.compiler.WinterProcessor",
        date = "2019-02-10T14:52Z"
)
public final class WithInjectedField_WinterMembersInjector implements MembersInjector<WithInjectedField> {
    @Override
    public void inject(final Graph graph, final WithInjectedField target) {
        target.field0 = graph.instanceOrNullByKey(new ClassTypeKey<String>(String.class, null));
    }
}

File: winter-compiler/src/test/resources/WithInjectedField_WinterMembersInjector.kt
package test

import io.jentz.winter.Graph
import io.jentz.winter.inject.MembersInjector
import javax.annotation.Generated
import kotlin.String

@Generated(
  value = ["io.jentz.winter.compiler.WinterProcessor"],
  date = "2019-02-10T14:52Z"
)
class WithInjectedField_WinterMembersInjector : MembersInjector<WithInjectedField> {
  override fun inject(graph: Graph, target: WithInjectedField) {
    target.field0 = graph.instanceOrNull<String>()
  }
}
